deb/publish/publish.sh

```diff
@@ -25,7 +25,7 @@ popd
 
 # merge the result
 pushd $D/binary
-  mvn org.kohsuke:apt-ftparchive-merge:1.4:merge -Durl="$DEB_URL/binary/" -Dout=../merged
+  mvn org.kohsuke:apt-ftparchive-merge:1.6:merge -Durl="$DEB_URL/binary/" -Dout=../merged
```

Comment:
The [changes](https://github.com/kohsuke/apt-ftparchive-merge/compare/apt-ftparchive-merge-1.4...apt-ftparchive-merge-1.6), since `apt-ftparchive-merge` does not have a changelog file.